pkg/controller/certificates/cleaner/cleaner.go

```diff
@@ -164,7 +164,7 @@ func isDeniedPastDeadline(csr *capi.CertificateSigningRequest) bool {
 func isFailedPastDeadline(csr *capi.CertificateSigningRequest) bool {
 	for _, c := range csr.Status.Conditions {
 		if c.Type == capi.CertificateFailed && isOlderThan(c.LastUpdateTime, deniedExpiration) {
-			klog.Infof("Cleaning CSR %q as it is more than %v old and failed.", csr.Name, deniedExpiration)
+			klog.InfoS("Cleaning CSR as it is past expiration and failed.", "csr", klog.KObj(csr), "deniedExpiration", deniedExpiration)
```

Comment:
```suggestion
		klog.InfoS("Cleaning CSR as it is past pending expiration and unhandled.", "csr", klog.KObj(csr), "age", time.Since(csr.CreationTime), "expiration": pendingExpiration, "condition", capi.CertificateFailed)
```

Reply:
I didn't use "age", time.Since(csr.CreationTime) as it throws this error: csr.CreationTime undefined (type *"k8s.io/api/certificates/v1".CertificateSigningRequest has no field or method CreationTime).